test/Serialization/PolyhedralGeometry.jl

```diff
@@ -3,92 +3,92 @@
     mktempdir() do path
         @testset "Graph" begin
             G = Graphs.complete_graph(4)
-            save(joinpath(path, "c4.graph"), G)
-            loaded = load(joinpath(path, "c4.graph"))
```

Comment:
One thing that is "lost" in this PR are these custom filenames. I did that intentionally, as they seemed to serve no purpose other than force me each time I add a test to think of a name. That said, if anyone things they should be kept, it wouldn't be a problem to adjust this PR to do so.

Reply:
Well, it wasn't as much the purpose to force you to think of something, but rather my ignorance of better solutions. :wink: